src/Optimizely/Event/Builder/EventBuilder.php

```diff
@@ -60,147 +59,174 @@ class EventBuilder
     ];
 
     /**
-     * @var array Associative array of parameters to be sent for the event.
-     */
-    private $_eventParams;
-
-    /**
-     * Helper function to reset event params.
-     */
-    private function resetParams()
-    {
-        $this->_eventParams = [];
-    }
-
-    /**
-     * @return array Params for the event.
-     */
-    private function getParams()
-    {
-        return $this->_eventParams;
-    }
-
-    /**
-     * Helper function to set parameters common to impression and conversion event.
+     * Helper function to get parameters common to impression and conversion event.
```

Comment:
Please check, what helper function you added here.

Reply:
- setCommonParams changed to getCommonParams. 
- resetParams and getParams have been removed, as they are not needed anymore